### PR TITLE
Create 2024-02-13.md

### PR DESCRIPTION
**Pull request by camronmartinez**
Release notes for the update to VSHE which  adds 2 restricted endpoints that will allow government agencies to search for Veterans using SSN

### CODE REVIEW

**Review by joeabraun — CHANGES_REQUESTED** [review]
Just added some small changes to reflect what is written in our currently published release notes.

File: content/verification/veteran-verification/release-notes/2024-02-13.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+We've added two restricted access endpoints that allow government agencies to search for Veterans using social security numbers (SSN), enhancing accessibility while ensuring data privacy.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-02-13.md:1]:
```suggestion
We've added two restricted access endpoints that allow government agencies to search for Veterans using social security numbers (SSNs), enhancing accessibility while ensuring data privacy.
```

File: content/verification/veteran-verification/release-notes/2024-02-13.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+* The endpoints replicate /status and /disability rating functionalities with the addition of using SSN to search.
+* Documentation was updated to reflect these additions and their restricted access.
+
+To learn more, see the [Veteran Service History and Eligibility v2 documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current)
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-02-13.md:5]:
```suggestion
To learn more, review the [Veteran Service History and Eligibility v2 documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current). 
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
A couple small changes, and should be good to go.

**Review by joeabraun — APPROVED** [review]
Looks good to me. UX approves!

File: content/verification/veteran-verification/release-notes/2024-02-13.md
Comment (melliroo) [inline content/verification/veteran-verification/release-notes/2024-02-13.md:1]: Confirmed w/ Lorri that "approved federal consumers" can replace "government agencies"

Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-02-13.md:1]: I've updated the wording to reflect "approved federal consumers"

**Review by melliroo — APPROVED** [review]

**Pull request merged**